bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -189,6 +190,19 @@ impl Script {
             && self.0[24] == OP_CHECKSIG.to_u8()
     }
 
+    /// Checks whether a script is push only.
+    #[inline]
+    pub fn is_push_only(&self) -> bool {
+        for inst in self.instructions() {
+            match inst {
+                Err(_) => return false,
+                Ok(Instruction::Op(_)) => return false,
+                Ok(Instruction::PushBytes(_)) => {}
+            }
+        }
```

Comment:
In 862e0f4349f308a93f0de7caf47116d577e15fdb:

This logic does not match Core. It does not consider the numeric opcodes nor OP_RESERVED to be pushes.

Reply:
Fixed.